pandas/tests/extension/test_masked.py

```diff
@@ -163,6 +163,16 @@ def data_for_grouping(dtype):
 
 
 class TestMaskedArrays(base.ExtensionTests):
+    @pytest.mark.parametrize("na_action", [None, "ignore"])
+    def test_map(self, data_missing, na_action):
+        result = data_missing.map(lambda x: x, na_action=na_action)
+        if data_missing.dtype == Float32Dtype():
```

Comment:
If there a solution to avoid this?

Reply:
Not with roundtripping through object, maybe your other suggestion above solves this. 

No if your suggestion doesn't work